src/CMakeLists.txt

```diff
@@ -50,6 +43,6 @@ endif()
 if (CMAKE_COMPILER_IS_GNUCC)
 set_property(TARGET mold PROPERTY C_STANDARD 99)
 set_property(TARGET mold PROPERTY C_STANDARD_REQUIRED ON)
```

Comment:
Should the standard be set to C99 on all platforms? Then on GNU you might enable extensions with `C_EXTENSIONS`?